bots/image-create

```diff
@@ -184,10 +188,13 @@ class MachineBuilder:
         data_file = os.path.join(data_dir, name)
         shutil.move(partial, data_file)
 
+        # Remove temp image file
+        os.unlink(self.machine.image_file)
```

Comment:
This is unnecessary if a real ```tempfile``` is used. Is there a reason not to use a temporary file?